app/app/urls.py

```diff
@@ -94,14 +94,17 @@
     url(r'^legal/cookie/?', dashboard.views.cookie, name='cookie'),
     url(r'^legal/prirp/?', dashboard.views.prirp, name='prirp'),
     url(r'^legal/apitos/?', dashboard.views.apitos, name='apitos'),
-    url(r'^funding/embed/?', dashboard.embed.embed, name='embed'),
-    url(r'^funding/avatar/?', dashboard.embed.avatar, name='avatar'),
     url(r'^profile/(.*)?', dashboard.views.profile, name='profile'),
     url(r'^toolbox/?', dashboard.views.toolbox, name='toolbox'),
     url(r'^tools/?', dashboard.views.toolbox, name='tools'),
     url(r'^gas/?', dashboard.views.gas, name='gas'),
     url(r'^coin/redeem/(.*)/?', dashboard.views.redeem_coin, name='redeem'),
 
+    # images
+    url(r'^funding/embed/?', dashboard.embed.embed, name='embed'),
+    url(r'^funding/avatar/?', dashboard.embed.avatar, name='avatar'),
+    url(r'^static/avatar/(.*)/(.*)?', dashboard.embed.avatar, name='avatar'),
```

Comment:
This will probably cause problems on reverse lookup.  Can we make these two url names unique and switch them over to use `re_path` ?